app/forms/hub/outbound_call_form.rb

```diff
@@ -46,35 +46,30 @@ def dial
       end
     end
 
+    # initiates a call to the from phone number when any number is pressed.
+    # If a number is pressed, the call will redirect to another twilio url that connects the call to the client
+    # After 5 seconds without a response the call will hang up.
     def twiml
       twiml = Twilio::TwiML::VoiceResponse.new
-      twiml.say(message: 'Please wait while we connect your call.')
-      # The status callback for the call is attached to the dial event to the client.
-      # This means that the length of the call will be based on how long the user was connected to the client,
-      # And the status will be based on whether the client picked up the call.
-      twiml.dial do |dial|
-        dial.number(@outbound_call.to_phone_number,
-                    status_callback_event: 'answered completed',
-                    status_callback: webhook_url,
-                    status_callback_method: 'POST')
+      twiml.gather(action: connect_to_client_url, num_digits: 1, timeout: 5) do
+        twiml.say(message: "Press any number to connect your Get Your Refund call.")
       end
+      # this code only executes if the user does not enter a digit within the timeout period (5 secs)
+      twiml.say(message: "We didn't hear from you so we're hanging up!")
+      twiml.hangup
       twiml.to_xml
     end
 
     private
 
-    def webhook_url
+    def connect_to_client_url
```

Comment:
consider making this some sort of shareable method since we're using the pattern in multiple places...

Reply:
test the behavior once there instead of multiple tests in different files for the same behavior.